server/api/spaceships.js

```diff
@@ -12,6 +13,18 @@ router.get('/', async (req, res, next) => {
   }
 });
 
+router.get('/featured', async (req, res, next) => {
+  try {
+    const featSpaceships = await Spaceship.findAll({
+      order: [Sequelize.literal('random()')],
```

Comment:
This was interesting! Didn't know you could order them straight from Sequelize. Good stuff

Reply:
Initially I had pulled all the data from the db into the component and was doing the randomization logic there, but then realized how inefficient it was and learned that you can pull random items out of the db with Sequelize! Pretty cool!